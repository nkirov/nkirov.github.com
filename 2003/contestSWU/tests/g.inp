5
21 3						
4 0 9 6					
16 4 19 20				
6 7 15 14
20 4
16 6 20 10
8 0 10 10
0 12 14 16
1 20 22 21
100 100
1 1 12 10
13 1 27 10
30 1 42 10
43 1 70 10
71 1 73 10
75 1 80 10
1 31 20 40
21 31 40 40
41 31 59 40
60 31 70 40
71 31 78 40
80 31 85 40
86 31 90 40
91 31 92 40
93 31 95 40
96 31 100 40 
88 1 90 10
91 1 92 10
93 1 100 10
1 11 10 20
11 11 20 20
21 11 30 20
31 11 40 20
41 11 50 20
51 11 60 20
61 11 70 20
71 11 80 20
82 11 90 20
91 11 99 20
1 21 5 30
6 21 12 30
13 21 20 30
21 21 29 30
30 21 39 30
40 21 50 30
51 21 62 30
63 21 75 30
76 21 89 30
90 21 100 30
1 42 2 45 
3 42 20 45
21 42 22 45
23 42 32 45
33 42 35 45
36 42 47 45
48 42 50 45
51 42 52 45
53 42 62 45
63 42 65 45
66 42 77 45
78 42 80 45
81 42 92 45
93 42 100 45 
1 48 15 60
17 48 23 60
28 48 44 60
48 48 50 60
59 48 73 60
80 48 88 60
90 48 96 60
1 62 10 71 
20 62 30 71
40 62 50 71
60 62 70 71
80 62 90 71
35 62 36 71
37 62 39 71
53 62 57 71
91 62 92 71
80 74 85 80
70 74 75 80
55 74 57 80
31 74 38 80
11 74 18 80
2 74 10 80
20 74 30 80
40 74 50 80
60 74 65 80
94 74 95 80
96 74 100 80
91 85 93 92
62 85 90 92
58 85 60 92
55 85 57 92
42 85 50 92
3 85 40 92
1 85 2 92
93 93 99 100
91 93 92 100
80 93 90 100
73 93 77 100
61 93 70 100
56 93 60 100
53 93 55 100
50 93 52 100
44 93 48 100
23 93 40 100
21 93 22 100
17 93 20 100
1 93 15 100
50 10
1 1 5 50
6 0 10 49
11 1 15 50
16 0 20 49
21 1 25 50
26 0 30 49
31 1 35 50
36 0 40 49
41 1 45 50
46 0 48 4
500 25
46 38 68 53
12 142 47 148
21 225 24 233
23 337 70 346
5 431 44 468
136 49 178 67
107 139 126 161
110 234 136 251
102 348 147 380
143 411 166 447
235 16 241 33
248 144 258 177
201 214 244 220
204 347 243 375
239 437 288 461
347 38 374 53
313 107 358 144
340 225 378 231
302 347 327 386
342 422 364 451
401 42 404 51
404 125 448 173
432 232 467 242
404 305 431 338
411 439 446 458