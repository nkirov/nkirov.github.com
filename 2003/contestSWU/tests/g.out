31.992470
29.760950
162.498823
470.436785
708.104955
